### ISSUE TITLE
Debian installation & dependencies problems

### ISSUE THREAD

**Issue by teymour** [bobthecow/genghis#231]
I had lot of problems to install genghis on a Debian (stretch).

The provided gem file does not install any dependencies. I had to install manually nearly all of them.

Here is the command I used : 

    sudo aptitude install ruby ruby-sinatra  ruby-mustache ruby-sinatra-contrib ruby-rack
    sudo gem install vegas -v 0.1.8
    sudo gem install sinatra -v 1.3.1
    sudo gem install sinatra-contrib -v 1.3.1
    sudo gem install sinatra-mustache -v 0.0.4
    sudo gem install mongo -v 1.12.5
    sudo gem install bson -v 1.12.5
    sudo gem install bson_ext -v 1.12.5
    sudo gem install genghisapp -v 2.3.11

**Comment by henkealg** [comment]
This project is abandoned, or so it seems. :|

**Comment by teymour** [comment]
Yes. I thought my solution could help others. This issue has more a documentation purpose than request for evolution ;)
